nntrainer/layers/conv2d_layer.cpp

```diff
@@ -186,20 +219,9 @@ static void im2col(const Tensor &in, const TensorDim &kdim,
   /// effective kernel width considering dilation
   unsigned int eff_k_width = (k_width - 1) * dilation[1] + 1;
 
-  unsigned int out_height = (height - eff_k_height) / mstride[0] + 1;
+  [[maybe_unused]] unsigned int out_height = (height - eff_k_height) / mstride[0] + 1;
   unsigned int out_width = (width - eff_k_width) / mstride[1] + 1;
```

Comment:
Is those two not used naymore? I guess we can delete?

Reply:
out_width is used at line 264. out_height is kept for completion purpose.

Reply:
I see, thanks